.github/workflows/buildkit.yml

```
@@ -38,4 +38,4 @@ jobs:
 
       - name: Build artifact
         if: github.ref == 'refs/heads/main'
-        run: packer build -var ami-name=depot-buildkit-2-snapshot buildkit/buildkit.pkr.hcl
+        run: packer build -var ami-name=depot-buildkit-3-snapshot buildkit/buildkit.pkr.hcl
```

Due to that limit on public AMIs that we haven't raised, we currently have to alternate between `-buildkit-` and `-buildkit-2-`. Someday we should automate this. 🙂 

```suggestion
        run: packer build -var ami-name=depot-buildkit-snapshot buildkit/buildkit.pkr.hcl
```